### PR TITLE
Courses and User Enum

### PR DESCRIPTION
**Pull request by Moonforeva**

### CODE REVIEW

**Comment by Moonforeva** [comment]
Ready

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  end
+
+  private
+
+    def course_params
+      params.require(:course).permit(:title, :description, :credits)
+    end
+
+    def find_course
+      @course = Course.find_by :id params[:id]
```
Comment (manhnt-0960) [inline app/controllers/courses_controller.rb]: let's handle case return nil

File: app/views/courses/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="col-md-6 col-md-offset-3">
+  <div class="new-course-form">
+		<h1>New Course</h1>
```
Comment (manhnt-0960) [inline app/views/courses/new.html.erb]: wrong indent

File: config/application.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Add configuration values here, as shown below.
```
Comment (manhnt-0960) [inline config/application.yml]: put this file to gitignore

**Comment by Moonforeva** [comment]
Ready

**Pull request merged**